lib/config/hackmdEnvironment.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
   allowAnonymous: toBooleanConfig(process.env.HMD_ALLOW_ANONYMOUS),
   allowAnonymousEdits: toBooleanConfig(process.env.HMD_ALLOW_ANONYMOUS_EDITS),
   allowFreeURL: toBooleanConfig(process.env.HMD_ALLOW_FREEURL),
+  requireFreeURLAuthentication: toBooleanConfig(process.env.HMD_REQUIRE_FREEURL_AUTH),
```

Comment:
This is not not needed. We don't add new config to a compatibly layer.

Reply:
Okay, removed. I honestly didn't even know what this file was good for, but I have assumed it's there for legacy reasons...